src/open_company/api/companies.clj

```diff
@@ -58,6 +62,37 @@
     ;; update the company
     {:updated-company (company/put-company slug (merge original-company patch-updates) user)}))
 
+;; ----- Validations -----
+
+(defn malformed-post-req?
```

Comment:
(Probably Schema could be utilized more to generate this information but this seemed fine too.)